src/include/access/xact.h

```diff
@@ -85,6 +85,7 @@ extern PGDLLIMPORT int synchronous_commit;
 /* used during logical streaming of a transaction */
 extern PGDLLIMPORT TransactionId CheckXidAlive;
 extern PGDLLIMPORT bool bsysscan;
```

Comment:
This can be removed, given this is the non-backport patch.

Reply:
OK